cmd/helm/install.go

```diff
@@ -212,6 +215,7 @@ func newInstallCmd(c helm.Interface, out io.Writer) *cobra.Command {
 	f.StringVar(&inst.version, "version", "", "specify the exact chart version to install. If this is not specified, the latest version is installed")
 	f.Int64Var(&inst.timeout, "timeout", 300, "time in seconds to wait for any individual Kubernetes operation (like Jobs for hooks)")
 	f.BoolVar(&inst.wait, "wait", false, "if set, will wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment are in a ready state before marking the release as successful. It will wait for as long as --timeout")
+	f.BoolVar(&inst.atomic, "atomic", false, "if set, installation process purges chart on fail")
```

Comment:
One last little thing here. We need to note in this documentation that the `--atomic` flag will automatically do a `--wait` so people know that is what happens. Same thing on the upgrade command

Reply:
Oh, sorry, I nearly forgot to do that :sweat_smile: Fixed that in latest commit